### ISSUE TITLE
Updated now getting "Entity not available: sensor.rpi4_8_powerstatus

### ISSUE THREAD

**Issue by boskoe** [Sennevds/system_sensors#58]
Hi,

I had installed in early August and system sensors was running well. Yesterday I tried updating to the latest code. I copied my settings.yaml to another directory. Deleted everything in the system_sensors directory then re ran the install instructions and copied my settings.yaml back into the src folder. Rebooted everything and now at the bottom of my Home Assistant Overview System Monitor card I get "Entity not available: sensor.rpi4_8_powerstatus.

I poked around a bit in MQTT Explorer and found this in the homeassistant branch:

> binary_sensor
F6416E_BTN_1
RPi4-8
RPi4-8PowerStatus
config = {"device_class":"problem","name":"RPi4-8 UnderVoltage","state_topic":"system-sensors/sensor/RPi4-8/state","value_template":"{{value_json.power_status}}","unique_id":"rpi4-8_sensor_power_status","availability_topic":"system-sensors/sensor/RPi4-8/availability","device":{"identifiers":["rpi4-8_sensor"],"name":"RPi4-8 Sensors","model":"RPI RPi4-8", "manufacturer":"RPI"}}
sensor
F6416E_status
CB96AD_status
CB8813_status
CB0967_status
RPi4-8
system-sensors
sensor
RPi4-8
availability = online
state = {"temperature": 45.0,"disk_use": 3.5,"memory_use": 11.0,"cpu_usage": 2.6,"swap_usage": 0.0,"power_status": "False","last_boot": "2020-10-02T09:41:36-04:00","last_message": "2020-10-03T08:58:32.056618-04:00", "updates": 0, "disk_use_drive1": 21.2}

I've exhausted my ability's of solving this. If you could give me any help I would greatly appreciate it.

Doug

**Comment by Sennevds** [comment]
The power status changed from a sensor to a binary sensor https://github.com/Sennevds/system_sensors/releases/tag/1.1.0

**Comment by boskoe** [comment]
I've install the zip file you indicated. I can see the binary sensor in the MQTT stream but Home Assistant card still indicates sensor not available. All the other parts work though.

**Comment by Sennevds** [comment]
Yeah because it's not a sensor anymore but a binsry_sensor so change sensor.rpi4_8_powerstatus into binary_sensor.rpi4_8_powerstatus in your lovelace config and it should work

**Comment by boskoe** [comment]
Okay this is really strange. When, as you instructed, I changed the sensor type in the card to "binary_sensor.rpi4_8_powerstatus" I still had the Entity not available message. I decided to look the Current Entities section HA's developer tools. The only binary sensor I have remotely related to this issue is "binary_sensor.rpi4_8_undervoltage". When I changed the card sensor to this value I have no errors. Card displays normally.

I still don't understand what has happened between your inductions and how my instance got the different binary sensor name.
I don't remember changing anything like that during installing and configuring.
